source/dmocks/mocks.d

```diff
@@ -772,6 +772,19 @@ unittest
     assert (calledPayload);
 }
 
+@("delegate payload with mismatching parameters")
+unittest
+{
+    Mocker r = new Mocker();
```

Comment:
(at least) three styles are mixed in the unit-test functions:
- the mocker is called `m`
- the mocker is called `r` (maybe due to a conflict with the mocked something that is called `m`)
- the mocker is called `mocker`

at least for new unit tests, I would prefer

    auto mocker = new Mocker();